PushNotification/PushNotification/content/portable-net45+wp8+wpa81+win8+MonoAndroid10+MonoTouch10+Xamarin.iOS10/Helpers/CrossPushNotificationListener.txt.pp

```diff
@@ -20,7 +20,7 @@
             Debug.WriteLine("Message Arrived");
         }
 
-        public void OnRegistered(string token, DeviceType deviceType)
+        public void OnRegistered(string Token, DeviceType deviceType)
         {
             Debug.WriteLine(string.Format("Push Notification - Device Registered - Token : {0}", Token));
```

Comment:
nice catch! I think is better to have it lower case and instead of changing the parameter, change:

Debug.WriteLine(string.Format("Push Notification - Device Registered - Token : {0}", Token));

for :

Debug.WriteLine(string.Format("Push Notification - Device Registered - Token : {0}", token));

I will merge and close this pull request.

Thanks